server/webapp/WEB-INF/rails.new/app/assets/javascripts/pipeline_configs/application.js

```diff
@@ -14,7 +14,13 @@
  * limitations under the License.
  */
 
-requirejs(['jquery', 'mithril', 'pipeline_configs/models/pluggable_tasks', 'pipeline_configs/views/pipeline_config_widget', 'foundation.topbar'], function ($, m, PluggableTasks, PipelineConfigWidget) {
+requirejs([
+  'jquery', 'mithril',
+  'pipeline_configs/models/pluggable_tasks', 'pipeline_configs/models/resources', 'pipeline_configs/models/users', 'pipeline_configs/models/roles',
+  'pipeline_configs/views/pipeline_config_widget', 'foundation.topbar'
+], function ($, m,
```

Comment:
requiring `foundation.topbar` registers itself in the jquery foundation namespace (somewhere) so it can be initialized via `$(document).foundation()`